isis/src/qisis/objs/CnetEditorView/CnetEditorView.h

```diff
@@ -51,11 +51,13 @@ namespace Isis {
    * @author 2018-04-04 Tracie Sucharski
    *
    * @internal
-   *    @history 2018-06-1 Kaitlyn Lee - Because AbstractProjectItemView now inherits
-   *                                from QMainWindow, I added a dummy central widget
-   *                                and set its layout to the grid layout. We used to set
-   *                                the whole CnetEditorView widget's layout, now we only
-   *                                set the central widget's layout.
+   *    @history 2018-06-01 Kaitlyn Lee - Because AbstractProjectItemView now inherits from QMainWindow,
+   *                              I added a dummy central widget and set its layout to the grid layout.
```

Comment:
If your history comment is multiple lines all lines after the first need to be indented 4 spaces after the start of your name. Meaning that the second line should line up with the l in your first name